fedscale/core/resource_manager.py

```diff
@@ -18,6 +18,11 @@ def register_tasks(self, clientsToRun):
         self.client_run_queue = clientsToRun.copy()
         self.client_run_queue_idx = 0
 
+    def get_remaining(self) -> int:
+        """Number of tasks left in the queue
+        """
+        return self.get_task_length()
```

Comment:
Maybe the get_task_length() function could be removed in favor of get_remaining?